safeeyes/config/lang/pl.json

```diff
@@ -0,0 +1,51 @@
+{
+    "meta_info": {
+        "language_name": "Polski",
+        "language_name_en": "Polish"
+    },
+    "exercises": {
+        "short_break_close_eyes": "Mocno zaciśnij powieki",
+        "short_break_roll_eyes": "Ruszaj oczami z boku na bok",
+        "short_break_rotate_clockwise": "Wykonuj krążenia oczami zgodnie z ruchem wskazówek zegara",
+        "short_break_rotate_counter_clockwise": "Wykonuj krążenia oczami przeciwnie do ruchu wskazówek zegara",
+        "short_break_blink": "Pomrugaj oczami",
+        "short_break_focus_far_distance": "Skup wzrok na odległym punkcie",
+        "short_break_drink_water": "Napij się wody",
+        "long_break_walk": "Przejdź się chwilę",
+        "long_break_lean_back": "Oprzyj się wygodnie na krześle i zrelaksuj"
+    },
+    "messages": {
+        "ready_for_a_break": "Przygotuj się! Przerwa za {} sekund(y).",
+        "disabled_until_restart": "Wyłączony do ponownego uruchomienia",
+        "disabled_until_x": "Wyłączony do {}",
+        "next_break_at": "Następna przerwa o {}"
+    },
+    "ui_controls": {
+        "about": "O programie",
+        "audible_alert": "Powiadomienie dźwiękowe na koniec przerwy",
+        "cancel": "Anuluj",
+        "disable": "Zatrzymaj Safe Eyes",
+        "enable": "Uruchom Safe Eyes",
+        "enable_screen_lock": "Zablokuj ekran po każdej długiej przerwie",
+        "for_x_hour": "Na {} godzinę",
+        "for_x_hours": "Na {} godzin(y)",
```

Comment:
This is always in the plural. I guess the brackets surrounding `y` is not necessary.